### PR TITLE
docs(agents-kit): Initial contribution of the 23.12 preview on the knowledge agents kit

### PR DESCRIPTION
**Pull request by drcgjung**
## Description

- Adds Agents Kit folder, pages and assets
- integrate into the kits sidebar and the preview carousel
- streamline documentation

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by danielmiehle** [comment]
Please make sure that all contributers have a valid ECA and resolve the conflicts. Thanks!

**Comment by drcgjung** [comment]
> Please make sure that all contributers have a valid ECA and resolve the conflicts. Thanks!

Thanks for the heads up, exactly that is the reason why that is a draft pull request that is not yet ready for review. We are working on the issues (ECA has been signed by the contributor AFTER the check did run).

File: docs-kits/kits/knowledge-agents/development-view/aas/bridge.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+formats from remote filesystems and APIs. This engine is used to build flat relational views onto a hierarchical
+json structure that may originate in the value-only-serialization of the AAS. Typically there will be one table/view
+per json-schema/submodel template. As an example, see these [scripts](https://github.com/catenax-ng/product-knowledge/tree/main/infrastructure/resources/dremio)
+* A graph engine (such as [ontop](https://ontop-vkg.org/guide/) ) that is able to bind/translate SPARQL queries into SQL. As an example, see these [bindings](https://github.com/catenax-ng/product-knowledge/tree/main/infrastructure/oem/resources/trace.obda)
+
+Of course, if the data is available in a native SQL-schema, the SQL/JSON-engine can be omitted. Likewise, even the graph engine
+can be left out if a sparql-capable database holds its data in conformance to the CX-ontologies.
+
+## KA-AAS Bridge
+
```
Comment (arnoweiss) [inline docs-kits/kits/knowledge-agents/development-view/aas/bridge.md:69]:
```md
Exposing substructures of the distributed knowledge graph via the AAS APIs is possible by deploying the [KA-AAS-Bridge](https://github.com/eclipse-tractusx/knowledge-agents-aas-bridge). This generic tool can be used to expose the graphs structures as AAS by configuring a set of mappings. Each consists of two components

* a SPARQL query extracting "flat" information out of the virtual graph
* a mapping configuration providing the basic structure of the target AAS

These two components must be closely coordinated with each other. The query is executed against an internal 
SPARQL-endpoint configured by the data provider. Its response (XML) is then digested by the [aas4j-transformation-library](https://github.com/eclipse-aas4j/aas4j-transformation-library) 
and transformed into AAS-native structures. This is executed at runtime whenever a request hits the AAS-APIs of the bridge so 
that the ground truth remains in the RDF-graph (or the persistence it was virtualized from, see above). [FAAAST framework](https://github.com/FraunhoferIOSB/FAAAST-Service/) provides 
the AAS-tooling required for the implementation of all relevant AAS-APIs. The library ships with four default mappings for 
Traceability-related Aspect-Models but is not restricted to these. Details on the KA-AAS-bridge's deployment can (soon) be found in [its documentation](https://github.com/eclipse-tractusx/knowledge-agents-aas-bridge/blob/main/README.md).
```

**Review by arnoweiss — APPROVED** [review]
Fine from the aas-bridge content pov

**Comment by maximilianong** [comment]
Hi @drcgjung - as a preparation for the meeting we have on Friday. @danielmiehle and me had a look into the KIT and just noticed some points for discussion:

- Changelog: please start the first PR with Version 0.1 and leave the snapshot information or put it together under one release. (Kits follow SemVer).
- The adoption view should follow our template (https://eclipse-tractusx.github.io/docs/kit-process/artefacts). That means, we see the page "Target Audience" under the headline of "Use Case". The FAQ could be part of the Business Value.

We had the question whether the Development Process page should be part of the general and overall TRGs of Tractus-X.

Regarding "Layers & Modules" - can this be menged with the AAS bridge (just one sub-page is not so nice).

**Comment by drcgjung** [comment]
Hi @maximilianong 

> * Changelog: please start the first PR with Version 0.1 and leave the snapshot information or put it together under one release. (Kits follow SemVer).

now have a 0.1 (scheduled for end of Sep) and a PRE 0.1 with all the other additions merged.

> * The adoption view should follow our template (https://eclipse-tractusx.github.io/docs/kit-process/artefacts). That means, we see the page "Target Audience" under the headline of "Use Case". The FAQ could be part of the Business Value.

the pages have been retitled/renamed.
 
> We had the question whether the Development Process page should be part of the general and overall TRGs of Tractus-X.

It was a requirement from the 23.09 Q-Gate, but if referring to TRGs is enough (or is obvious for all the KITs from the context) I would be happy to erase it.

> Regarding "Layers & Modules" - can this be menged with the AAS bridge (just one sub-page is not so nice).

done.

**Comment by drcgjung** [comment]
Discussed aligned with @maximilianong
- Adoption view pages merged into single page.
- Testbed moved to operation view
- Development process deleted (in favor to surrounding developer hub documentation)

**Pull request merged**